.github/workflows/spring-oracle-example.yml

```
@@ -0,0 +1,68 @@
+        with:
+          compose-file: './spring-oracle-example/compose.yaml'
+          services: ${{ steps.services.outputs.services }}
+          up-flags: '--build'
+          down-flags: '--volumes'
+
+      - name: Wait for containers to initialize
+        run: sleep 20
+
+      - name: Check container health
+        run: |
+          ./.github/scripts/check-container-health.sh "${{ steps.services.outputs.services }}"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)